src/easylink/utilities/general_utils.py

```diff
@@ -34,14 +35,7 @@ def wrapped(*args, **kwargs):
 
 
 def configure_logging_to_terminal(verbose: int):
-    """Sets up logging to ``sys.stdout``.
-
-    Parameters
```

Comment:
Any particular reason you removed this?

Reply:
Uhh, I don't remember. I think I meant to just change "Parameters" to "Arguments"